### DISCUSSION TITLE
Do we have these functions?

### DISCUSSION THREAD

**Discussion by probonopd** [probonopd/MiniDexed#92] [Q&A] [answered]
Do we have these functions? Should they be added to the menu?

> POLY/MONO
> Determines whether the instrument will function in the polyphonic or monophonic mode. 
> The range of the portamento effect is different in the polyphonic and monophonic modes.
> 
> PITCH BEND
> Two keys are used to determines the effect of the PITCH BEND thumbwheel.
> 
> RANGE: 
> The range of pitch bend can be set from 0 to 12. 0 range is equivalent to no pitch bend. A setting of 12 permits pitch bend over a plus minus 1200 cent (2 octave) range, If the range is set at 7, then pitch bend will be possible over a plus minus 700 cent range (i.e. plus or minus one fifth).
> 
> STEP:
> The step parameter can be set from 0 to 12. A setting of 0 corresponds to 0-cent steps, and a setting of 12 corresponds to 1200-cent (1 octave) steps. If STEP is set to 0, then a perfectly smooth pitch hand will result. IF STEP is set to 1, the pitch will bend in 100-cent (semitone) steps. 
> 
> PORTAMENTO
> The portamento effete varies according to whether the instrument is in the polyphonic or monophonic mode. 
> 
> MONOPHONIC MODE: 
> In this mode press the DATA ENTRY [minus 1] key to activate "FINGERED PORTA." In this mode portamento is applied only to legato notes. 
> Press the DATA ENTRY [plus 1] key to activate "FULL TIME PORTA." In this mode portamento is always applied. 
> 
> POLYPHONIC MODE: 
> "SUS-KEY P RETAIN." In this mode the pitch of keys released while the sustain pedal is on or of notes that have a long sustain time does not change. However, portamento is effected between two subsequently pressed keys. 
> "SUS-KEY P FOLLOW."" In this mode the pitch of a key released while the sustain pedal is held slides (portamento) to a previously pressed key. There is no change with continuously pressed keys. 
> 
> GLISSANDO: 
> The glissando function is turned either ON or OFF. When it is OFF a normal portamento effect is produced. 
> 
> TIME: 
> Adjusts the speed of the portamento/glissando effect from 0 to 99. A0 setting results in no effect, while a setting of 99 produces the longest (slowest) portamento or glissando.

**Comment by dcoredump** [comment, marked as answer]
They are all implemented (and they work on MicroDexed). Only the portamento code has a small bug in combination with glissando (which I cannot find). Also protamento is not fully working in mono mode.

Ah, I am a little bit wrong. I think MONOPHONIC MODE and POLYPHONIC MODE are not implemnted...
